_unittest/test_01_Design.py

```diff
@@ -230,3 +230,6 @@ def test_24_change_type(self):
 
     def test_25_change_registry_from_file(self):
         assert self.aedtapp.set_registry_from_file(os.path.join(local_path, "example_models", "Test.acf"))
+
+    def test_27_odesktop(self):
+        assert self.aedtapp.odesktop
```

Comment:
Should be some sort of `isinstance` here. If you can't get a type, this works as well:
```py
assert 'COM' in `str(type(self.aedtapp.odesktop))`
```
Note: I "COM" probably isn't the string to test.

Reply:
@akaszynski Excellent idea. Done. I will try to improve other existing tests too in an other PR.